src/Stripe/Services/Invoices/StripeUpcomingInvoiceLinesOptions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Stripe
+{
+    public class StripeUpcomingInvoiceLinesOptions : StripeListOptions
+    {
+        [JsonProperty("coupon")]
+        public string Coupon { get; set; }
+
+        [JsonProperty("subscription")]
+        public string Subscription { get; set; }
+
+        [JsonProperty("subscription_plan")]
+        public string SubscriptionPlan { get; set; }
+
+        [JsonProperty("subscription_prorate")]
+        public string SubscriptionProrate { get; set; }
+
+        [JsonProperty("subscription_proration_date")]
+        public int? SubscriptionProrationDate { get; set; }
+
+        [JsonProperty("subscription_quantity")]
```

Comment:
Looks like some of this has gone stale since it's taken me awhile to get to these - this is now quantity.  And many other properties are missing.